website/src/components/index-page.tsx

```diff
@@ -28,8 +26,6 @@ export function IndexPage(): ReactElement {
         }}
       />
 
-      <LiveDemo />
```

Comment:
This did not add any value in its current state IMHO as it does not show the benefits of client preset.